astromon/db.py

```diff
@@ -515,7 +519,7 @@ def set_formats(dat):
             dat[col.name].info.format = fmts.get(col.name, '.2f')
 
 
-def get_cross_matches(name='astromon_21', dbfile=None):
+def get_cross_matches(name='astromon_21', dbfile=None, **kwargs):
```

Comment:
This function should not include `**kwargs` since as stated it is for getting a standard cross match. The problem is that the `**kwargs` filtering might be inconsistent with the `name` filter and give unexpected results (e.g. setting `snr` smaller than what is in `astromon_21`).

Reply:
You should expect to get results in the standard sample **and** `snr >= 1.0` and that is not inconsistent.

Reply:
But the API or documentation does not say anywhere what is the standard cross match, so how as a user am I supposed to know what is there?  

Anyway, I'm saying that as a user it is totally *CONFUSING* that there are two functions which can be used to get filtered cross matches, but one of them starts with an implicit set of filter criteria that is not obvious.

The function documentation itself says that if you want your own set of filter parameters then use `compute_cross_matches`.
```
    If you want *non-standard* cross-match, with your own set of parameters, refer to the
    :any:`compute_cross_matches <cross_match.compute_cross_matches>` function.
```

So each function should do exactly one thing, and this is all that is needed:
- `get_cross_matches` gets a standard cross match by name
- `compute_cross_matches` computes a custom cross match

I will also say that for me as a user this result is completely unexpected:
```
In [11]: xc = get_cross_matches(snr=1.0)

In [12]: np.min(xc['snr'])
Out[12]: 3.0013847
```
As a developer who has spent hours reading the code I understand it but that does not change my initial reaction as a user.

Reply:
OK, I'm rescinding this comment, leave the API as it is.